nova-app/src/shared/ui/input/ChipInput.tsx

```
@@ -53,31 +58,35 @@ export const ChipInput = ({
       .filter((v) => v && !value.includes(v));
-    if (newChips.length) onChange([...value, ...newChips]);
-    setInputValue('');
-  };
 
-  const addChip = (chipValue?: string) => {
-    const finalValue = chipValue ?? inputValue;
-    if (!finalValue) return;
-    addChipsFromValue(finalValue);
+    if (newChips.length) {
+      onChange([...value, ...newChips]);
+      newChips.forEach((chip) => onAdd?.(chip));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)